common/lib/capa/capa/templates/optioninput.html

```diff
@@ -30,6 +30,6 @@
   </div>
   <p class="answer" id="answer_${id}"></p>
   % if msg:
-      <span class="message">${HTML(msg)}</span>
+      <span class="message" aria-describedby="label_${id}" aria-live="polite" tabindex="-1">${HTML(msg)}</span>
```

Comment:
It's not clear to me if aria-describedby should point to the label (as you have here), or the select (which is what the status is doing on line 27. @cptvitamin can you offer some guidance? In general we have been pointing to the "inputs" in the other templates.